setup.cfg

```diff
@@ -1,6 +1,21 @@
+[bdist_wheel]
+universal=1
+
 [metadata]
+license_file = COPYING.LESSER
 description-file = README.markdown
 
+[check-manifest]
+ignore =
+  bin/*
+  dev_tool/*
+  test/fixture/cli/vital.vim/*
```

Comment:
Then `.gitmodules` is not needed also (which holds only this).

Reply:
I was thinking it is useful to include .gitmodules so sdist consumers know there is another set of test data they need to get.

But another approach is to make the test skip if the data doesnt exist, and emit a message about it.  iirc test_cli_vital passes when the git submodule has not been fetched.

Reply:
I've fixed the test in https://github.com/Kuniwak/vint/pull/320 - now it would be skipped.

I think this is fine then - include .gitmodules, but not any checked out submodule.